.github/workflows/publish-latest.yml

```diff
@@ -5,6 +5,17 @@ on: workflow_dispatch
 jobs:
   publish-npm-latest:
     uses: ./.github/workflows/publish-npm-latest.yml
+    secrets:
+      CAP_GH_RELEASE_TOKEN: ${{ secrets.CAP_GH_RELEASE_TOKEN }}
+      NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
   publish-android:
     needs: publish-npm-latest
-    uses: ./.github/workflows/publish-android.yml
\ No newline at end of file
+    uses: ./.github/workflows/publish-android.yml
+    with:
```

Comment:
does this need to be `secrets:` as well?

Reply:
Yes it should be secrets

Reply:
fixed